kinto/plugins/admin/views.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+
+from pyramid.settings import aslist
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+RESPONSE_CACHED = None
+
+
+# Configured home page
+def admin_home_view(request):
+    global RESPONSE_CACHED
```

Comment:
It's been a long time I've not been doing any Python but it's odd we need a global here...